### DISCUSSION TITLE
Project 5: Better interfacing of Blender and MDAnalysis

### DISCUSSION THREAD

**Discussion by abhishekd358** [MDAnalysis/mdanalysis#4510] [GSoC Discussions]
@BradyAJohnston
@yuxuanzhuang

Hello, mentors

my name is Abhishek and I wanted to contribute to this project. Please, may I know how should I involve with this project.

**Comment by hmacdope** [comment]
Check our GSOC guide here: https://github.com/MDAnalysis/mdanalysis/wiki/Google-Summer-Of-Code

**Reply by abhishekd358** [reply]
Thank you for the sharing me the information. I'll definitely go through it.

**Reply by hmacdope** [reply]
Mark as resolved if you have what you need.
